backend/src/bundles/users/user.service.ts

```diff
@@ -19,6 +20,30 @@ class UserService implements Service {
         return Promise.resolve(null);
     }
 
+    public async findByEmail(
+        email: UserEntity['email'],
+    ): Promise<UserGetOneItemResponseDto> {
+        const user = await this.userRepository.findByEmail(email);
+
+        if (!user) {
+            throw new Error('Incorrect email');
```

Comment:
Change new Error to any exception or create new in folder /shared/src/framework/exceptions